### PR TITLE
Added language Northern Kurdish (Kurmanjî)

### LINKED ISSUE #2529: Kurdish (Kurmanji) (kmr)
* Language name:  Kurdish (Kurmanji) 
* Language info: https://en.wikipedia.org/wiki/Kurmanji
* ISO 639-3 code: KMR
* Public List: https://tatoeba.org/eng/sentences_lists/show/168672
* Icon: https://imgur.com/a/aYPRlaz
* Requested by: https://tatoeba.org/eng/user/profile/Sarchia

Note: We're checking which members have posted sentences in this language.

### PR DESCRIPTION
**Pull request by RyckRichards**
This PR closes #2529.
It adds Kurdish (Kurmanji), one of the languages that will replace Kurdish (kur).

### CODE REVIEW

File: src/Lib/LanguagesLib.php [hunk 1]
```diff
@@ -612,6 +612,7 @@ public static function languagesInTatoeba()
                 'mni' => __d('languages', 'Meitei'),
                 'ayl' => __d('languages', 'Libyan Arabic'),
                 'ood' => __d('languages', "O'odham"),
+                'kmr' => __d('languages', 'Kurdish (Kurmanji)',
```
Comment (jiru) [inline src/Lib/LanguagesLib.php:615]: You forgot a closing parenthesis.

**Comment by soliloquist-tatoeba** [comment]
There are still red error marks on the PRs of the Kurdish languages.

**Comment by AndiPersti** [comment]
They are marked with an error because the file changed since opening the PRs and so they need to be updated (which is easy to do but [I'm still not sure about the exact procedure](https://github.com/Tatoeba/tatoeba2/issues/2540#issuecomment-698999651)).

**Comment by trang** [comment]
@RyckRichards Please change the name to `Northern Kurdish (Kurmanjî)` as suggested in https://github.com/Tatoeba/tatoeba2/issues/2529#issuecomment-699597267.

**Pull request merged**